graphrag_eval/steps/evaluation.py

```diff
@@ -1,41 +1,49 @@
 import json
 from collections import defaultdict
+from typing import Any
+from collections.abc import Sequence
 
 from .retrieval_context_ids import recall_at_k
 from .sparql import compare_sparql_results
 
 
-def compare_steps_outputs(reference: dict, actual: dict) -> float:
-    ref_output = reference.get("output")
-    act_output = actual["output"]
-    assert ref_output, "Reference step output is mandatory"
-    if reference.get("output_media_type") == "application/sparql-results+json":
+Match = tuple[int, int, int, float]
+Step = dict[str, Any]
+ReferenceGroup = Sequence[Step]
```

Comment:
Great :+1:

Reply:
Below we have `actual_steps: Sequence[Step],` , so do you think we can rename this to `StepsGroup` of `Steps` or something similar and use it there as well?